code/game/objects/items.dm

```diff
@@ -587,7 +587,7 @@ var/global/image/fire_overlay = image("icon" = 'icons/effects/fire.dmi', "icon_s
 		var/turf/T = get_turf(src)
 		var/obj/effect/decal/cleanable/ash/A = new()

```

Comment:
the desc doesn't matter if it's insta killed
you can't read it in the literally miliseconds this occurs in.